SwiftAdvanceExercise.playground/Pages/Class_1.1.xcplaygroundpage/Contents.swift

```diff
@@ -0,0 +1,20 @@
+// MARK: Create a swift program to demonstrate swift class. Which has two variables inside class body. Access (Set its values and get its value.) these variables by creating one instance of class.
+
+class Project {
```

Comment:
class ClassName {
keep a line
// code
keep a line
}
same goes for all the files
keep in mind, this line spacing do not apply to functions